### DISCUSSION TITLE
The name “Dialect” is misleading

### DISCUSSION THREAD

**Discussion by fitojb** [dialect-app/dialect#199] [Ideas]
I just think it’s a bit ironic that I can’t e.g. translate to Brazilian or European or Angolan Portuguese, accounting for the vocabulary and grammar differences in each ;-)

**Comment by rafaelmardojai** [comment]
So the point of opening the issue is?
Do you want to suggest a name change? Or request the creation of translation models for different language dialects in LibreTranslate?

Maybe we need to create some chat room to avoid this kind of issue tickets ;-)

**Comment by fitojb** [comment]
Sure, feel free to move this to a GitHub Discussion instead. :+1: 

Sorry for being unclear. I think the name could use some improvement, certainly. It’s not great to suggest that the app does something that it’s currently out of reach for most machine translation systems, so something less misleading would be very helpful. And I don’t know much about LibreTranslate, but if it can avoid duplicating work by inheriting some linguistic rule logic from e.g. Apertium and LanguageTool, that’d be great.

**Comment by rafaelmardojai** [comment]
If you want to know how we ended up with "Dialect" you can check this issue: #29.

I really don't think that "Dialect" is misleading, from my perspective it says nothing, it's just a pretty name related to linguistics. You need to read the description to understand what it does: `Translate between languages` / `A translation app for GNOME`. 

More than out of reach, I would say that is unnecessary, most people can understand what is said in their dialect's mother language, more useful is translating from a dialect to a complete different language and Google for example already supports that.

Apertium is rule-based while LibreTranslate (which is a web API wrapping Argos Translate) uses neural machine translation, but I guess some work can be can be recycled.

**Comment by fitojb** [comment]
> from my perspective it says nothing, it's just a pretty name related to linguistics

This is very telling. Well yes, you guys made an unfortunate choice on that ticket, I’m sorry to tell you. There was a reason why no other app was ever called this.

**Reply by rafaelmardojai** [reply]
Well, works for me. We liked it as the name of the app that we develop on our free time, and give for free. 

It's a nice name from my pov. It can be improved? Of course! But please keep the constructive dialogue, give us some alternatives if you have them.

The way you started the discussion was not even respectful.

**Comment by mufeedali** [comment]
I completely agree with @rafaelmardojai . The way you brought this up was "odd", to say the least.

Past the initial stage of development, where a few people decide together on a name, it's not usually worth it to go back and rethink the naming. Especially so in the case of Dialect and other GNOME or GNOME Circle apps, where we follow a set of guidelines for the naming.

I would've been totally fine with it if, for example, our naming had unfortunately offended someone or if, at least, more appropriate alternatives are provided. 

Rebranding is more troublesome than most people realise and is generally not worth it. Where it is worth it though, it is welcomed. For example, if a suggested naming is a huge improvement over the existing one or if the current naming offends someone. So please keep these things in mind going forward :)
